Cargo.toml

```diff
@@ -7,28 +7,44 @@ license = "MIT OR Apache-2.0"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [features]
 default = ["serde", "async"]
-serde = ["dep:serde", "hex", "serde_json"]
 async = ["tokio", "tokio-stream", "async-stream"]
 spec-tests = ["serde", "serde_yaml"]
 gen-spec = ["syn", "prettyplease", "quote"]
 gen-tests = ["walkdir", "convert_case"]
 
+std = [
+    "rand/std",
+    "sha2/std",
+    "multihash/std",
+    "bs58/std",
+    "getrandom/std",
+    "serde/std",
+    "ssz-rs/std"
+]
+
+serde = ["dep:hex", "serde_json"]
+
 [dependencies]
-ssz-rs = { git = "https://github.com/ralexstokes/ssz-rs", rev = "cb08f1" }
-blst = "0.3.6"
-rand = "0.8.4"
-thiserror = "1.0.30"
-sha2 = "0.9.8"
-integer-sqrt = "0.1.5"
-enr = "0.6.2"
+getrandom = { version = "0.2.8", default-features=false, features = ["js"] }
+ssz-rs = { git = "https://github.com/Snowfork/ssz_rs", branch="feat/contribution", default-features=false }
+rand = {version = "0.8.4", default-features = false}
+thiserror = {version = "1.0.30",  optional=true}
+error-chain={version = "0.12.4", default-features=false}
+sha2 = { version ="0.9.8", default-features = false }
+integer-sqrt = {version = "0.1.5", default-features = false }
+enr = {version = "0.6.2" }
 multihash = { version = "0.16", default-features = false, features = ["std", "multihash-impl", "identity", "sha2"] }
-multiaddr = "0.14.0"
+multiaddr = {version="0.14.0" }
+hashbrown = {version="0.13.1"}
 
-serde = { version = "1.0", features = ["derive"], optional = true }
+serde = { version = "1.0", default-features = false, features = ["derive"] }
 serde_json = { version = "1.0.81", optional = true }
 serde_yaml = { version = "0.8", optional = true }
 hex = {version = "0.4.3", optional = true }
 
+blst = {version = "0.3.6", optional=true}
+milagro_bls = { git = "https://github.com/webb-tools/milagro_bls", default-features = false }
```

Comment:
why nor use the original milagro_bls, it's latest version supports no-std https://github.com/sigp/milagro_bls